app/Http/Controllers/CharacterController.php

```diff
@@ -28,7 +28,7 @@ public function __construct()
     private function getValidationRules() {
         return [
             'member_id'     => 'nullable|integer|exists:members,id',
-            'name'          => 'nullable|string|min:2|max:32',

```

Comment:
You may want to put [required](https://laravel.com/docs/7.x/validation#rule-required) where `nullable` was. I can't recall off the top of my head if it acts as required when neither `nullable` nor `required` are present, but it's probably best to be distinct.

Unless I'm misunderstanding the intention of removing `nullable`, and you actually _don't_ want to add `required` for some reason.

Reply:
I updated it to required, it would work either way. Just different error feedback for the user. 

Example Error without required: ![image](https://user-images.githubusercontent.com/69526309/90271043-0c929f00-de10-11ea-9667-309f48f39ae4.png)

Example Error with required: 
![image](https://user-images.githubusercontent.com/69526309/90271282-66936480-de10-11ea-885e-4c92d09f9f2c.png)